frontend/routes/auth/wallets/key-details/accounts-store.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import { RpcMethods } from '../../../../lib/client-rpc-methods'
+import { useAccountsStore } from './accounts-store'
+
+const ACCOUNT_FIXTURE = {
```

Comment:
We might end up needing data like this shared at some point but will revisit when we nail down a long term mocking solution for e2e.

Reply:
Completely agree, but out of scope for this IMO 👍

Reply:
oh yeah absolutely